int32 operator_id
string decision_source
